services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -337,8 +371,9 @@ private CommandMessage buildDeleteFlowCommand(final String id, final String corr
      */
     private CompletableFuture<BatchResults> flowPushUnpush(List<FlowInfoData> externalFlows, FlowOperation op) {
         final String correlationId = RequestCorrelationId.getId();
+        String collect = externalFlows.stream().map(FlowInfoData::getFlowId).collect(Collectors.joining());
```

Comment:
Maybe `Collectors.joining(", ")`? Otherwise all flowId will be printed without delimiter.